_sass/components/_layout.scss

```diff
@@ -116,3 +116,19 @@ article.container,
   @include u-margin-top(8);
   @include u-padding-top(4);
 }
+
+#touchpoints_feedback {
+  margin-top: 2rem;
+}
+
+#fba-modal-title, .section-title-view {
+  display: none;
+}
+
+.usa-button.usa-button-group__item {
+  color: $button-blue;
+  background-color: $white;
+  border: 2px solid $button-blue;
```

Comment:
Since `$button-blue` is the same as `$blue`, do we need a new variable, or can we use it instead?

```suggestion
  color: $blue;
  background-color: $white;
  border: 2px solid $blue;
```